nvflare/recipe/poc_env.py

```diff
@@ -11,17 +11,307 @@
+                examples_dir=None,  # No examples directory
+            )
+            print(f"POC workspace prepared successfully at: {self.poc_workspace}")
+
+        except Exception as e:
+            raise RuntimeError(f"Failed to prepare POC workspace: {e}")
+
+    def _start_poc_services(self):
+        """Start POC services (server, clients) excluding admin console."""
+        try:
+            print("Starting POC services (excluding admin console)...")
+            _start_poc(
+                poc_workspace=self.poc_workspace,
+                gpu_ids=self.gpu_ids,
+                excluded=["admin@nvidia.com"],  # Exclude admin console like --ex admin@nvidia.com
+                services_list=[],
+            )
+            print("POC services started successfully")
+
+            # Give services time to start up
+            time.sleep(3)
+
+        except Exception as e:
+            raise RuntimeError(f"Failed to start POC services: {e}")
+
+    def _stop_poc_services(self):
+        """Stop POC services (excluding admin console)."""
+        try:
+            print("Stopping POC services...")
+            _stop_poc(
+                poc_workspace=self.poc_workspace,
+                excluded=["admin@nvidia.com"],  # Exclude admin console (consistent with start)
+                services_list=[],
+            )
+            self._poc_started_by_us = False
+            print("POC services stopped successfully")
+
+        except Exception as e:
+            print(f"Warning: Failed to stop POC services cleanly: {e}")
+
+    def _clean_poc_workspace(self):
+        """Clean up POC workspace by removing all generated files."""
+        try:
+            print(f"Cleaning POC workspace: {self.poc_workspace}")
+            _clean_poc(self.poc_workspace)
+            print("POC workspace cleaned successfully")
+
+        except Exception as e:
+            print(f"Warning: Failed to clean POC workspace: {e}")
+
+    def _submit_job_via_api(self, job_path: str) -> str:
+        """Submit job via Flare API to POC environment.
+
+        Args:
+            job_path: Path to the exported job directory.
+
+        Returns:
+            str: Job ID returned by the system.
+        """
+        try:
+            # Get the admin startup kit path for POC
+            admin_dir = self._get_admin_startup_kit_path()
+
+            # Create secure session with POC admin
+            sess = new_secure_session(
+                username="admin@nvidia.com",  # Default POC admin user
+                startup_kit_location=admin_dir,
+            )
+
+            try:
+                # Submit the job
+                job_id = sess.submit_job(job_path)
+                return job_id
+            finally:
+                sess.close()
+
+        except Exception as e:
+            raise RuntimeError(f"Failed to submit job via Flare API: {e}")
+
+    def _monitor_job(self, job_id: str, duration: int):
```

Comment:
You can use the same sess created when submitting the job. Don't create another sess.

Reply:
thanks, changed as suggested